.github/workflows/unit-tests.yml

```diff
@@ -82,33 +82,19 @@ jobs:
           runVcpkgInstall: true
           # Release 2023.04.15
           vcpkgGitCommitId: '501db0f17ef6df184fcdbfbe0f87cde2313b6ab1'
-      - name: Patch vcpkg library headers with patchelf
-        run: |
-          cd vcpkg
-          ${{ github.workspace }}/.github/workflows/vcpkg-fix-rpaths
-          cd ..
-      - uses: actions/setup-python@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-          cache: 'pip'
-      - name: Upgrade pip to v23+
-        run: |
-          pip install --upgrade pip
-          pip --version
-          python --version
-      - name: Run build+install with pip
+      - name: Run CMake build
         run: |
-          SKBUILD_CONFIGURE_OPTIONS=\
-          " -DCMAKE_TOOLCHAIN_FILE=${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"\
-          " -DVCPKG_TARGET_TRIPLET=x64-linux-dynamic" pip install -v .
+          mkdir build
+          cd build
+          cmake \
+            -DCMAKE_TOOLCHAIN_FILE="${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake" \
```

Comment:
It looks like this workflow was disabled, but should this now specify `UPSP_BUILD_TESTING=ON` as well?

Reply:
Thank you for the catch. Yes will re-enable this workflow and force-set cmake opts (now that by default they are OFF)